main.go

```diff
@@ -31,17 +34,22 @@ func main() {
 	err := fSetup.Initialize()
 	if err != nil {
 		fmt.Printf("Unable to initialize the Fabric SDK: %v\n", err)
+		return
 	}
 
 	// Install and instantiate the chaincode
 	err = fSetup.InstallAndInstantiateCC()
 	if err != nil {
 		fmt.Printf("Unable to install and instantiate the chaincode: %v\n", err)
+		return
 	}
 
 	// Launch the web application listening
 	app := &controllers.Application{
 		Fabric: &fSetup,
 	}
 	web.Serve(app)
+
+	// Close SDK
+	fSetup.CloseSDK()
```

Comment:
use a `defer` just after the SDK init

Reply:
The SDK is init in setup.go, if I would defer right after its init in the Initialize() function it would be destroyed at the end of this function. 
One idea could be to make it public and defer it from the main function however I don't think that it's a good idea to make it public only to defer it.
What do you think ?

Reply:
Change this please